aws-amplify/amplify-ui
# feat(geo): Geo component end to end tests

#### Description of changes

This Pull Request adds the end to end tests for `AmplifyMap` and `Geocoder` components to our feature branch (`feat-geo`) which will be merged to `main` on the date of the scheduled GA release for Geo components.

This pull request is part of a series:
- https://github.com/aws-amplify/amplify-ui/pull/1570
- https://github.com/aws-amplify/amplify-ui/pull/1571
- https://github.com/aws-amplify/amplify-ui/pull/1573
- https://github.com/aws-amplify/amplify-ui/pull/1574





#### How to validate tests
1. Pull our geo tag branch which contains all components and tests
```shell
git fetch origin && git checkout ui-geo/main
```
2. Pull the map test environment
```shell
cd environments/basic-map && yarn pull && cd -
```
3. Setup the package
```shell
yarn setup
```
4. Start a development instance of the example apps
```shell
yarn next-example dev
```
5. In a new terminal window, run the geo e2e tests and verify they pass
```shell
yarn e2e test:geo -e "TAGS=@react and not @skip"
``` 

#### Checklist




By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
